src/theme.ts

```diff
@@ -121,7 +121,7 @@ export const weights = [400, 700];
 export const radius = 3;
 
 export const font = `'Nunito', Arial, sans-serif`;
```

Comment:
Let's make this SSP as well

Reply:
@sradevski should we keep both `font` and `titleFont`?

Reply:
@StefKors yeah let's keep it for now, it doesn't really add that much overhead anyway

Reply:
+1